File: Udemy/Algoritmos e Logica de Programacao/Java/ExerciciosJavaEstruturaCondicional/src/application/ProblemaLanchonete.java
package application;

import java.util.Locale;
import java.util.Scanner;

public class ProblemaLanchonete {

	public static void main(String[] args) {
		/*
		 * Problema "lanchonete" (adaptado de URI 1038)
Uma lanchonete possui vários produtos. Cada produto possui um código
e um preço. Você deve fazer um programa para ler o código e a
quantidade comprada de um produto (suponha um código válido), e daí
informar qual o valor a ser pago, com duas casas decimais, conforme
tabela de produtos ao lado.
Código do produto 					Preço do produto
1										 R$ 5.00
2 										 R$ 3.50
3 										 R$ 4.80
4 										 R$ 8.90
5 										 R$ 7.32
		 * */
		
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		

		int codigo, qtd;
		double preco;
		System.out.print("Codigo do produto comprado: ");
		codigo = sc.nextInt();
		preco = 0.0;
		qtd = 0;
		if(codigo == 1) {
			preco = 5.00;
		}else if(codigo == 2) {
			preco = 3.50;
		}else if(codigo == 3) {
			preco = 4.80;
		}else if(codigo == 4) {
			preco = 8.90;
		}else if(codigo == 5) {
			preco = 7.32;
		}
		
		System.out.print("Quantidade comprada");
		qtd = sc.nextInt();
		
		System.out.print("Valor a pagar: R$ "+String.format("%.2f",preco * qtd));
		
		sc.close();

	}

}

File: Algoritmos e Logica de Programacao/C/Estrutura Condicional/ProblemaLanchonete.c
/*
Problema "lanchonete" (adaptado de URI 1038)
Uma lanchonete possui vários produtos. Cada produto possui um código 
e um preço. Você deve fazer um programa para ler o código e a 
quantidade comprada de um produto (suponha um código válido), e daí 
informar qual o valor a ser pago, com duas casas decimais, conforme 
tabela de produtos ao lado.
*/

#include <stdio.h>
#include <stdlib.h>

int main (){
	
	int codigo, qtd;
	double produto, valor;


	printf("Codigo do produto comprado: ");
	scanf("%d",&codigo);
	
	if(codigo == 1)
	{
		produto = 5.00;
	}else if(codigo == 2)
	{
		produto = 3.50;
	}else if(codigo == 3)
	{
		produto = 4.80;
	}else if(codigo == 4)
	{
		produto = 8.90;
	}else if(codigo == 5)
	{
		produto = 7.32;
	}
	
	printf("Quantidade comprada: ");
	scanf("%d",&qtd);
	
	valor = produto * qtd;
	
	printf("Valor a pagar: %.2lf", valor);
	
	return 0;
	
}
